prow/prstatus/prstatus_test.go

```diff
@@ -90,117 +115,230 @@ func TestServeHTTPWithoutLogin(t *testing.T) {
 	if err := yaml.Unmarshal(body, &dataReturned); err != nil {
 		t.Errorf("Error with unmarshaling response: %v", err)
 	}
-	if !reflect.DeepEqual(dataReturned, mockData) {
+	if !equality.Semantic.DeepEqual(dataReturned, mockData) {
```

Comment:
`equality.Semantic` is useful for k8s types and adds a level of semantic meaning to the checks, which isn't (?) appropriate for these types. was `reflect` not working?

Reply:
I have some problems with reflect before that I could not remember what they were. Maybe I was wrong, I will use reflect instead

Reply:
Fixed